### ISSUE TITLE
Is it possible to set the number of audio channels using Transformer?

### ISSUE THREAD

**Issue by hortemo** [androidx/media#388]
Hi! Is it possible to set the number of audio channels using Transformer? E.g. change the audio in a video from stereo to meno? Thanks!

**Comment by kim-vde** [comment]
Yes, you can use the [`ChannelMixingAudioProcessor`](https://github.com/androidx/media/blob/main/libraries/common/src/main/java/androidx/media3/common/audio/ChannelMixingAudioProcessor.java), which has just been released in the 1.1.0-alpha01 release.

Assuming you use 1.1.0-alpha01 (tell me if it's not the case), to apply an `AudioProcessor`, you need to:
- Create an [`Effects`](https://github.com/androidx/media/blob/1.1.0-alpha01/libraries/transformer/src/main/java/androidx/media3/transformer/Effects.java) instance that contains this `AudioProcessor`.
- Create an [`EditedMediaItem`](https://github.com/androidx/media/blob/1.1.0-alpha01/libraries/transformer/src/main/java/androidx/media3/transformer/EditedMediaItem.java) instance with this `Effects` instance.
- Pass this `EditedMediaItem` to [`Transformer.start()`](https://github.com/androidx/media/blob/1.1.0-alpha01/libraries/transformer/src/main/java/androidx/media3/transformer/Transformer.java#L757)
